### PR TITLE
new-clip-on-paste behavior pref, false by default

### LINKED ISSUE #5521: Create preference for paste-as-clip vs paste-into-clip
Located in: Preferences > Track Behaviors > Behaviors

Option name: "Always paste audio as new clips"

Option either restores pre-3.4 behavior or keeps 3.4 behavior. The old behavior is the default

### PR DESCRIPTION
**Pull request by saintmatthieu**
Resolves: #5521

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: src/prefs/TracksBehaviorsPrefs.cpp [hunk 1]
```diff
@@ -101,9 +101,9 @@ void TracksBehaviorsPrefs::PopulateOrExchange(ShuttleGui & S)
       S.TieCheckBox(XXO("Enable &dragging selection edges"),
                     {wxT("/GUI/AdjustSelectionEdges"),
                      true});
-      S
-         .TieCheckBox(XXO("Editing a clip can &move other clips"),
+      S.TieCheckBox(XXO("Editing a clip can &move other clips"),
             EditClipsCanMove);
+      S.TieCheckBox(XXO("Ask Leo"), { wxT("/GUI/AskLeo"), false });
```
Comment (saintmatthieu) [inline src/prefs/TracksBehaviorsPrefs.cpp:106]: @LWinterberg awaiting exact wording.

**Comment by Paul-Licameli** [comment]
This draft looks right so far, of course needing real strings

**Comment by saintmatthieu** [comment]
There is still one tiny behaviour difference, which I think is rather a relief, though (but it also could be that my judgment is biased by laziness since it'd also mean more code changes and more risk of introducing new bugs):
Screenshot 2023-11-07 at 18 10 18

3.3.3: paste clipboard data onto selection, on either track, and the not-enough-room error pops up.
3.4.2 (with new setting off): paste clipboard data onto selection on
* upper track: the rightmost clip moves right.
* lower track: the not-enough-room error pops up.

So it'd be more lenient. @LWinterberg what do you think?

File: src/prefs/TracksBehaviorsPrefs.cpp (hunk 1 above)
Comment (LWinterberg) [reply src/prefs/TracksBehaviorsPrefs.cpp]: "Always paste audio as new clips"

**Comment by LWinterberg** [comment]
LGTM!

**Review by Paul-Licameli — APPROVED** [review]

**Comment by saintmatthieu** [comment]
... cherry-picked onto 3.4.2.

**Pull request merged**